Limits/python/Models.py

```diff
@@ -371,6 +376,271 @@ def build(self,ws,label):
+                   ws.arg(a1Name), ws.arg(n1Name), ws.arg(a2Name), ws.arg(n2Name) )
+        self.wsimport(ws, doubleCB)
+        self.params = [meanName,sigmaName,a1Name,n1Name,a2Name,n2Name]
+
+class DoubleCrystalBallSpline(ModelSpline):
+
+    def __init__(self,name,**kwargs):
+        super(DoubleCrystalBallSpline,self).__init__(name,**kwargs)
+        if not hasattr(ROOT,"DoubleCrystalBall"):
+            ROOT.gROOT.ProcessLine(".L /afs/cern.ch/work/k/ktos/public/Plotting/CMSSW_8_1_0/src/CombineLimits/Limits/macros/DoubleCrystalBall.cpp")
+
+    def build(self,ws,label):
+        print "CHECK1"
+        logging.debug('Building {}'.format(label))
+        meanName = 'mean_{0}'.format(label)
+        sigmaName = 'sigma_{0}'.format(label)
+        a1Name = 'a1_{0}'.format(label)
+        n1Name = 'n1_{0}'.format(label)
+        a2Name = 'a2_{0}'.format(label)
+        n2Name = 'n2_{0}'.format(label)
+        masses = self.kwargs.get('masses', [])
+        means  = self.kwargs.get('means',  [])
+        sigmas = self.kwargs.get('sigmas', [])
+        a1s = self.kwargs.get('a1s', [])
+        n1s = self.kwargs.get('n1s', [])
+        a2s = self.kwargs.get('a2s', [])
+        n2s = self.kwargs.get('n2s', [])
+        print "CHECK2"
+        # splines
+        meanSpline  = ROOT.RooSpline1D(meanName,  meanName,  ws.var('MH'), len(masses), array('d',masses), array('d',means))
+        sigmaSpline = ROOT.RooSpline1D(sigmaName, sigmaName, ws.var('MH'), len(masses), array('d',masses), array('d',sigmas))
+        a1Spline =     ROOT.RooSpline1D(a1Name, a1Name, ws.var('MH'), len(masses), array('d',masses), array('d',a1s))
+        n1Spline =     ROOT.RooSpline1D(n1Name, n1Name, ws.var('MH'), len(masses), array('d',masses), array('d',n1s))
+        a2Spline =     ROOT.RooSpline1D(a2Name, a2Name, ws.var('MH'), len(masses), array('d',masses), array('d',a2s))
+        n2Spline =     ROOT.RooSpline1D(n2Name, n2Name, ws.var('MH'), len(masses), array('d',masses), array('d',n2s))
+        print "CHECK3"
+        # import
+        getattr(ws, "import")(meanSpline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(sigmaSpline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(a1Spline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(n1Spline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(a2Spline, ROOT.RooFit.RecycleConflictNodes())
+        getattr(ws, "import")(n2Spline, ROOT.RooFit.RecycleConflictNodes())
+        print "CHECK4"
+
+        # build model
+        doubleCB = ROOT.DoubleCrystalBall(label, label, ws.arg(self.x), ws.arg(meanName), ws.arg(sigmaName), 
+                   ws.arg(a1Name), ws.arg(n1Name), ws.arg(a2Name), ws.arg(n2Name) )
+        print  "DOUBLECV", doubleCB.Print()
+        self.wsimport(ws, doubleCB)
+        self.params = [meanName,sigmaName,a1Name,n1Name,a2Name,n2Name]
+
+class DoubleGaussian(Model):
```

Comment:
Call it `TwoSidedGaussian` or similar. Not `DoubleSided`, since we are using `Double` to indicate two overlapping functions with the same mean (ie `DoubleCrystalBall`).